apps/website/src/pages/docs/datepicker.mdx

```diff
@@ -22,36 +22,64 @@ import * as AllExamples from '~/examples';
 
 When a user needs to schedule deadlines, timeframes, or for age verification purposes, we provide them with a date picker. It behaves much like a select menu. We currently have three different types of date pickers.
 
-## Appearance
+## Variants
 
-### Default
+### Basic
 
-The default date picker is a default menu button from which an interactive calendar expands on click. This allows the user to visually select the date rather than entering the date with a keyboard. This has the advantage of also showing weekdays, which may be important in a given context. The date picker is commonly used within column filters.
+The basic date picker consists of a default menu button from which an interactive calendar expands on click.
 
-### Text field
+<LiveExample src='DatePicker.basic.tsx' truncate={false}>
```

Comment:
`truncate={false}` means that the code takes full space it needs, we usually do that only on main example. Other code examples should be truncated.

Reply:
Fixed.